src/bridge/bridgemain.h

```diff
@@ -1149,6 +1149,7 @@ typedef enum
     GUI_DISASSEMBLY,
     GUI_DUMP,
     GUI_STACK,
+    GUI_THREADS,

```

Comment:
You cannot insert enums for binary compatibility. Please append it to the end

Reply:
Appended to the end.